app/cli/internal/action/attestation_push.go

```diff
@@ -133,7 +134,7 @@ func (action *AttestationPush) Run(ctx context.Context, attestationID string, ru
 
 	wrappedSigner := signer.GetSigner(action.keyPath, action.Logger, pb.NewSigningServiceClient(action.CPConnection))
 	renderer, err := renderer.NewAttestationRenderer(action.c.CraftingState, action.cliVersion, action.cliDigest, wrappedSigner,
-		renderer.WithLogger(action.Logger))
+		renderer.WithLogger(action.Logger), renderer.WithBundle(action.bundlePath))
```

Comment:
this naming is a little bit cofusing to me, it feel like you are providing a bundle when what you are saying is that it should output somewhere right?

In that case I'd call it `WithBundleOutput`, or `WithBundleOutputPath` wdyt?